scss/styles.scss

```diff
@@ -0,0 +1,95 @@
+$vars-nav-height: 90px;
+$vars-footer-height: 30vh;
+$vars-triangle-heigth: 90px;
+$vars-triangle-width: 100px;
+$vars-tech-logo-size: 100px;
+$vars-font-big: 24px;
+$vars-font-medium: 16px;
+$vars-font-small: 12px;
+
+* {
+  font-family: Roboto;
```

Comment:
add a fallback font* similar to `Roboto` so it loads when Roboto hasn't loaded yet.

something like this: ` font-family: 'Roboto', sans-serif;`

*: fallback fonts are fonts preloaded in the browser to use, there are very few and most of them are ugly. But sans-serif is a really good one and is similar to Roboto.